src/styles/blocks/calendar.scss

```
@@ -0,0 +1,38 @@
+.calendar {
+  @include flex-positioning(flex-start, center);
+  flex-wrap: wrap;
+  gap: $day-box-gap;
+  width: $calendar-width;
+
+  @for $var from 28 through 31 {
+    &--month-length--#{$var} :nth-child(n + #{$var + 1}) {
+      display: none;
+    }
+  }
+
+  @each $day, $num in $days {
+    &--start-date--#{$day} :first-child {
+      margin-left: ($num - 1) * ($day-box-size + $day-box-gap);
+    }
+  }
+
+  &__day {
+    @include flex-positioning(center, center);
+    box-sizing: border-box;
+    width: $day-box-size;
+    height: $day-box-size;
+    border: 1px solid #000;
+    transition: transform 0.5s;
+
```

```suggestion
    transition: transform 0.5s;
    background-color: #fff;

```
It's better to declare a default value and then change it on :hover...